.env.test

```
@@ -4,3 +4,7 @@ EARTHLY_INSTALLATION_NAME=translate
 OTEL_SERVICE_NAME=translate
 OTEL_EXPORTER_OTLP_INSECURE=true
 OTEL_RESOURCE_ATTRIBUTES=deployment.environment=test
+
+TRANSLATE_SERVICE_PORT=8080
+TRANSLATE_SERVICE_ADDRESS=localhost
```

In general, the address is used for <host>:<port>. Use HOST instead.

```suggestion
TRANSLATE_SERVICE_HOST=localhost
```